### ISSUE TITLE
Conversion to .onnx format?

### ISSUE THREAD

**Issue by abubakar2141731** [JialeCao001/D2Det-mmdet2.1#5]
Hi, 
the script for converting the model to onnx format is not working , some packages were not yet implemented, can you offer a work around for this problem?

**Comment by JialeCao001** [comment]
@abubakar2141731 Do you mean that convert the model in D2Det to D2Det-mmdet2.1?

**Comment by abubakar2141731** [comment]
i am using D2Det-mmdet 2.1, in the tools section there is a script named
pytorch2onnx.py, D2Det is not compatible with this script.

**Comment by JialeCao001** [comment]
Yeah. Maybe. I just train the model using D2Det-mmdet2.1. What is the function of pytorch2onnx.py/.

**Comment by abubakar2141731** [comment]
it converts a pytorch model to .onnx format to be deployed on a cpu.
